repose-aggregator/artifacts/docker/README.adoc

```diff
@@ -0,0 +1,126 @@
+= Repose Docker Images
+
+This document explains how to build a local image based on the `Dockerfile` 's located in the `src` directory.
+
+If you are instead just looking to use a pre-built docker image, then check us out on Docker Hub.
+
+image::http://dockeri.co/image/rackerlabs/repose[Repose Docker,link=https://hub.docker.com/r/rackerlabs/repose/]
+
+== Setting up the environment
+
+Whether the intent is to simply test the tasks or to actually build a local image, the Docker Container Engine needs to be running.
+Since these instructions are fairly involved, they are outside the scope of this document.
+However, they can be found https://docs.docker.com/engine/installation/[here].
+
+At this point you may be able to execute the `docker run hello-world` test.
+If you can, then you are almost ready to create a local Repose Docker image.
+If you can't, then the steps below may get you up and going.
+Either way, you will need to perform them.
+
+== Minor updates
+
+These are a few minor configuration items that may prevent some platforms from even executing the `docker run hello-world` test.
+They are not always well documented in the installation instructions, but are easily remedied.
+
+You will need to get a https://hub.docker.com[Docker Hub] account.
+Once you have that account, you will need to login using:
+
+- `docker login`
+
+Once you have confirmed your credentials, you will need to add them to the `~/.gradle/gradle.properties` file to allow them to work with the Repose build:
```

Comment:
Not technically true. You could also pass them as properties when executing Gradle. Pedantic, I know, but Gradle does its normal property lookup stuff.

Reply:
Ok.